+ soulbound/QF/PoPP info to DID page (#6683)
ethereum/ethereum-org-website: src/content/decentralized-identity/index.md

Identity underpins virtually every aspect of your life today. Using online services, opening a bank account, voting in elections, buying property, securing employment—all of these things require proving your identity.

> However, traditional identity management systems have long relied on centralized intermediaries who issue, hold, and control your identifiers and [attestations](#what-are-attestations). This means you cannot control your identity-related information, or decide who has access to personally identifiable information (PII) and how much access these parties have.

Reviewer: ```suggestion
However, traditional identity management systems have long relied on centralized intermediaries who issue, hold, and control your identifiers and [attestations](#what-are-attestations). This means you cannot control your identity-related information or decide who has access to personally identifiable information (PII) and how much access these parties have.
```

3. The smart contract selling shares can check the registry contract for the identities of screened buyers, making it possible for the smart contract to determine who is permitted to buy shares or not.

> ### Soulbound tokens and identity

Reviewer: ```suggestion
### Soulbound tokens and identity {#soulbound}
```

> [Soulbound Tokens](https://vitalik.ca/general/2022/01/26/soulbound.html) (non-transferable NFTs) could be used to collect information unique to a specific wallet. This effectively creates a unique on-chain identity bound to a specific Ethereum address that could include tokens representing achievements (e.g. finishing some specific online course or a particular score in a game) or participation in communities.

Reviewer: ```suggestion
[Soulbound tokens](https://vitalik.ca/general/2022/01/26/soulbound.html) (non-transferable NFTs) could be used to collect information unique to a specific wallet. This effectively creates a unique on-chain identity bound to a particular Ethereum address that could include tokens representing achievements (e.g. finishing some specific online course or passing a threshold score in a game) or community participation.
```

4. Decentralized identity makes identity data portable. Users store attestations and identifiers in mobile wallet and can share with any party of their choice. Decentralized identifiers and attestations are not locked into the database of the issuing organization.

> 5. Decentralized identity should work well with emerging zero-knowledge technologies that will enable individuals to prove they own or have done something without revealing what that thing is. This could become a powerful way to combine trust and privacy for applications such as voting.
> 
> 6. Decentralized identity enables anti-Sybil mechanisms that can identify when one individual human is pretending to be multiple humans in order to game or spam some system.

Reviewer: ```suggestion
5. Decentralized identity should work well with emerging zero-knowledge technologies that will enable individuals to prove they own or have done something without revealing what that thing is. This could become a powerful way to combine trust and privacy for applications such as voting.

6. Decentralized identity enables anti-Sybil mechanisms to identify when one individual human is pretending to be multiple humans to game or spam some system.
```

Decentralized identity can help create online communities that are free of fake accounts. For example, each user might have to authenticate their identity using an on-chain identity system, like the Ethereum Name Service, reducing the possibility of bots.

> ### 4. Anti-Sybil protection

Reviewer: ```suggestion
### 4. Anti-Sybil protection {#sybil-protection}
```

> Sybil attacks refer to individual humans tricking a system into thinking they are multiple people in order to increase their influence. [Grant-giving applications](https://gitcoin.co/grants/) that use [quadratic voting](https://www.radicalxchange.org/concepts/plural-voting/) are vulnerable to these Sybil attacks because the value of a grant is increased when more individuals vote for it, incentivizing users to split their contributions across many identities. Decentralized identities help to prevent this by rasing the burden on each participant to prove that they are really human, although often without having to reveal specific private information.

Reviewer: ```suggestion
Sybil attacks refer to individual humans tricking a system into thinking they are multiple people to increase their influence. [Grant-giving applications](https://gitcoin.co/grants/) that use [quadratic voting](https://www.radicalxchange.org/concepts/plural-voting/) are vulnerable to these Sybil attacks because the value of a grant is increased when more individuals vote for it, incentivizing users to split their contributions across many identities. Decentralized identities help to prevent this by raising the burden on each participant to prove that they are really human, although often without having to reveal specific private information.
```

- **[SpruceID](https://www.spruceid.com/)** - _A decentralized identity project which allows users to control digital identity with Ethereum accounts and ENS profiles instead of relying on third-party services._
- **[Proof of Humanity](https://www.proofofhumanity.id)** - _Proof of Humanity (or PoH) is a social identity verification system built on Ethereum._
- **[BrightID](https://www.brightid.org/)** - _A decentralized, open-source social identity network seeking to reform identity verification through the creation and analysis of a social graph._

> - - **[Proof-of-personhood Passport](https://proofofpersonhood.com/)** - _A decentralized digital identity aggregator._

Reviewer: ```suggestion
- **[Proof-of-personhood Passport](https://proofofpersonhood.com/)** - _A decentralized digital identity aggregator._
```
